src/styles/schedule.scss

```diff
@@ -1,72 +1,137 @@
-.audience {
+@mixin textFlex {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   font-family: "Roboto";
   font-size: 20px;
+}
 
-  &__type {
-    background-color: $mint;
-    border-radius: 40px;
-    margin-top: 1rem;
-    padding: 0.5rem 1rem;
+.schedule {
+  &__tab {
+    @media (max-width: $lg) {
+      display: inline-flex;
+      flex: 1;
+      // need !important to override default bootstrap padding on tabs
+      padding-left: 0 !important;
+      padding-right: 0 !important;
+      justify-content: center;
+      align-items: center;
+    }
   }
 
-  &__company {
-    background-color: $light-purple;
-    border-radius: 40px;
-    margin: 1rem;
-    padding: 0.5rem 1rem;
+  &__table {
+    background-color: #fff;
+    margin-top: -2px;
   }
-}
 
-.category {
-  display: flex;
-  flex-direction: column;
-  font-family: "Roboto";
-  font-size: 20px;
-
-  &__time {
-    margin-bottom: 0;
+  &__row {
+    display: flex;
   }
 
-  &__type {
-    margin-bottom: 0;
+  &__label {
+    padding: 0.2rem !important;
   }
-}
 
-.label {
-  padding: 0.2rem !important;
-}
+  &__arrow {
+    display: flex;
+    align-items: center;
+    &__icon {
+      width: 50px;
+      @media (max-width: $lg) {
+        width: 20px;
+      }
+    }
+  }
 
-.location {
-  display: flex;
-  flex-direction: column;
-  font-family: "Roboto";
-  font-size: 20px;
+  &__responsive {
+    display: flex;
+    flex: 3;
+    flex-direction: row;
+    @media (max-width: $md) {
+      flex-direction: column-reverse;
+    }
+  }
 
-  &__title {
-    font-weight: 600;
-    margin-bottom: 0;
+  &__category {
+    padding-right: 1.5rem;
+    flex: 1;
+    @include textFlex;
+    @media (max-width: $md) {
+      padding-right: 0;
+      justify-content: space-between;
+      flex-direction: row;
+    }
+    &__time {
+      margin-bottom: 0;
+    }
+    &__type {
+      margin-bottom: 0;
+    }
+    // the longest a time can possibly be
+    min-width: 200px;
   }
 
-  &__zoom {
-    color: #000;
-    margin-bottom: 0;
+  &__location {
+    flex-shrink: 0;
+    flex: 2;
+    @include textFlex;
+    @media (max-width: $md) {
+      justify-content: space-between;
+      flex-direction: row;
+    }
+    &__title {
+      max-width: 50vw;
+      white-space: nowrap;
```

Comment:
I think should prob do wrap? Cuz right now I think the u can't really see the word since u don't have the scrolling enabled (or maybe im wrong) so like people can only see "..." if the title is too long

Reply:
Yeet!